.circleci/config.yml

```diff
@@ -52,14 +51,24 @@ jobs:
       - checkout-and-dependencies
       - run: yarn flow:ci
 
-  build:
+  docker:
     executor: node
     steps:
       - checkout-and-dependencies
-      - run: yarn build:clean build
+      # This sets up a remote environment that's necessary to run docker commands.
+      - setup_remote_docker
+      - run:
+          name: "Build the Docker image"
+          command: yarn docker:build --pull --build-arg circle_build_url=$CIRCLE_BUILD_URL
+      - run:
+          name: "Start the docker image in a container and check it's running"
+          command: |
+            yarn docker:run:detached
+            # Wait up to 10 seconds that the server is launched.
+            timeout 10s sh \<<'EOF'
+            while ! docker exec profiler-server curl --silent --show-error --fail http://localhost:8000/__version__ ; do
+              sleep 1
+            done
```

Comment:
This is a bit ugly but I couldn't find a better solution.
`curl --retry` didn't seem to work properly in this situation :/

Reply:
Weird, this seems to work here: https://github.com/mozilla-services/poucave/blob/77704cb9e11a8ef2e1775b7449f9a09fc87d381e/.circleci/config.yml#L46

Reply:
yeah, to be more precise, it was working locally but not on circleci. Maybe some timing problem :/
In the end I used this shell snippet, a bit more verbose, but at least this always works.